eg2.py

```diff
@@ -0,0 +1,41 @@
+"""You are given a list of person names. Your task is to find out the three most frequent and three least frequent names."""
+
+
+names = ['Smith', 'Johnson', 'Williams', 'Jones', 'Brown', 'Davis', 'Miller', 'Wilson', 'Moore', 'Taylor', 'Anderson', 'Thomas', 'Jackson', 'White']   #created list with person names  
+
+print('Name List=',names)      #here we will see the list of persons 
+
+LengthOfItems=[]     #created empty list for store the len of each element
+
+#below empty lists created for store the equal length
+Length5=[]  #item with length 5
+Length6=[]  #item with length 6
+Length7=[]  #item with length 7
+Length8=[]  #item with length 8
+
```

Comment:
This becomes static code. Let's say if I have list of 10000 names and then I run this code, then it will not actually return the correct result. So, let's modifies this code to make it dynamic in a way that it can handle most of the cases and any length of dictionary with any number of names.